Gruntfile.js

```diff
@@ -40,7 +40,7 @@ module.exports = function(grunt) {
     umd: {
       main: {
         options: {
-          src: '<%= concat.dist.src %>',
+          src: '<%= concat.dist.dest %>',
           dest: 'dist/wavesurfer.umd.js',
```

Comment:
why is wavesurfer.umd.js still created? Shouldn't this be wavesurfer.min.js? If not, can we get rid of it? wavesurfer.dev.js is our non-minified reference build file now isn't it?